sqswatcher/sqswatcher.py

```diff
@@ -110,35 +110,22 @@ def _setup_ddb_table(region, table_name, proxy_config):
     :param proxy_config: proxy configuration
     :return: the Table object
     """
-    log.debug('running _setup_ddb_table')
+    log.debug("running _setup_ddb_table")
 
-    dynamodb = boto3.client('dynamodb', region_name=region, config=proxy_config)
-    tables = dynamodb.list_tables().get('TableNames')
+    dynamodb = boto3.client("dynamodb", region_name=region, config=proxy_config)
+    tables = dynamodb.list_tables().get("TableNames")
 
-    dynamodb2 = boto3.resource('dynamodb',  region_name=region, config=proxy_config)
+    dynamodb2 = boto3.resource("dynamodb", region_name=region, config=proxy_config)
     if table_name in tables:
         _table = dynamodb2.Table(table_name)
     else:
         _table = dynamodb2.create_table(
             TableName=table_name,
-            KeySchema=[
-                {
-                    'AttributeName': 'instanceId',
-                    'KeyType': 'HASH'
-                }
-            ],
-            AttributeDefinitions=[
-                {
-                    'AttributeName': 'instanceId',
-                    'AttributeType': 'S'
-                }
-            ],
-            ProvisionedThroughput={
-                'ReadCapacityUnits': 5,
-                'WriteCapacityUnits': 5
-            },
+            KeySchema=[{"AttributeName": "instanceId", "KeyType": "HASH"}],
```

Comment:
It is not part of your patch but I think we should remove this code, because we are creating the table in the CFN template

Reply:
As agreed we'll handle this as part of a separate PR